iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/ScalingParameters.java

```diff
@@ -34,6 +48,10 @@ public class ScalingParameters {
 
     private boolean allowsGeneratorOutOfActivePowerLimits = DEFAULT_ALLOWS_GENERATOR_OUT_OF_ACTIVE_POWER_LIMITS;
 
+    private DistributionMode distributionMode = null;
```

Comment:
I think there could be a default value that would be `REGULAR_DISTRIBUTION`, what do you think?